Dockerfile

```diff
@@ -39,37 +39,41 @@ RUN mkdir /opt/kimai && \
 #fpm alpine php extension base
 FROM php:7.3.10-fpm-alpine3.10 AS fpm-alpine-php-ext-base
 RUN apk add --no-cache \
-    # gd
-    libpng-dev \
-    freetype-dev \
-    # icu
-    icu-dev \
-    # zip
-    libzip-dev \
     # build-tools
-    m4 \
-    perl \
     autoconf \
     dpkg \
     dpkg-dev \
-    libmagic \
     file \
-    make \
-    re2c \
-    libgomp \
+    g++ \
+    gcc \
     libatomic \
-    mpfr3 \
+    libc-dev \
+    libgomp \
+    libmagic \
+    m4 \
+    make \
     mpc1 \
-    gcc \
+    mpfr3 \
     musl-dev \
-    libc-dev \
-    g++
+    perl \
+    re2c \
+    # gd
+    freetype-dev \
+    libpng-dev \
+    # icu
+    icu-dev \
+    # ldap
+    icu-dev \
+    openldap-dev \
```

Comment:
is this the only new dep?
after the reordering its not as easy to see

Reply:
yep